# Scripts and commands used for the Mimulus TR evolution study

## K-seek based k-mer analysis
### K-seek script was downloaded from the k-seek github
https://github.com/weikevinhc/k-seek

### slurm script that used k-seek script for analyzing FASTQ
KSEEK.slurm

### commands to calculate average coverage of samples. This will be used for normalizing k-mer counts
genomecov.sh

### commands for conducting trinity analysis
trinity_commands.sh
